default_config_file

```diff
@@ -81,9 +81,9 @@ list_slow_probes_only = False
 print_summary_stats = False
 ;
 ; Which item to return from the split-list. First element is 0.
-dns_response_item_occurence_to_return = 1
+dns_response_item_occurrence_to_return = 1
```

Comment:
This is an API break. I'd expect it to be rejected as is, although, I'd for it to be accepted as is.

Happy to drop it/handle it separately.

Reply:
Hello,

Thank you for sending those spelling corrections over, apologies for the delay in getting back to you.

If you could remove the variable name change, that would be great, as i will need to spend some time on solutions for working with existing config. files with the old spelling.

Also, the default_config_file has chanegd since your original PR -- it has a a new variable, and i implemented a few corrections of your corrections while i was editing it, so probably best to regenerate a new PR for the comments to avoid the merge conflict, if you don't mind?

Thanks, johan

Reply:
I've done a rebase and validated against my tooling, the only things it has an opinion on are this item (which I've dropped) and a python function (which I'd ignore if I were updating my baseline).